lib/portlayer/network/network.go

```diff
@@ -94,8 +74,13 @@ func Init(ctx context.Context, sess *session.Session, source extraconfig.DataSou
 		}
 
 		var netctx *Context
-		netctx, err = NewContext(*bridgeRange, *bridgeWidth, &config)
+		netctx, err = NewContext(ctx, sess, *bridgeRange, *bridgeWidth, &config)
 		if err != nil {
+			log.Warn(err)
```

Comment:
well, previous it didn't return anything. I don't want to have those errors swallowed without even one message out.